plugins/@grouparoo/bigquery/src/lib/table-import/getPropertyValues.ts

```diff
@@ -23,7 +23,7 @@ export const getPropertyValues: GetPropertyValuesMethod = async ({
   let responses: { [key: string]: { [column: string]: DataResponse[] } } = {};
   let aggSelect = columnNames.map((col) => `\`${col}\``).join(", ");
   let orderBy = "";
-  let groupByColumns = [tablePrimaryKeyCol];
+  let groupByColumns = [...new Set(tablePrimaryKeyCol)];
```

Comment:
💥

Reply:
Bigquery is the only one who had these shenanigans... so I only changed Bigquery.